src/together/utils/files.py

```diff
@@ -61,17 +61,22 @@ def check_file(
     if file.suffix == ".jsonl":
         report_dict["filetype"] = "jsonl"
         data_report_dict = _check_jsonl(file)
+
+        report_dict.update(data_report_dict)
+
     elif file.suffix == ".parquet":
         report_dict["filetype"] = "parquet"
         data_report_dict = _check_parquet(file)
+
+        report_dict.update(data_report_dict)
+
     else:
         report_dict["filetype"] = (
             f"Unknown extension of file {file}. "
             "Only files with extensions .jsonl and .parquet are supported."
         )
         report_dict["is_check_passed"] = False
 
-    report_dict.update(data_report_dict)
```

Comment:
Good catch! I would personally just create an empty `data_report_dict` for the unknown filetype to guarantee that the `update` always happens regardless of the branch in the code (there once was a bug where we missed this at some point afaik)

Reply:
updated!